src/openrct2/paint/Supports.h

```diff
@@ -93,6 +93,7 @@ enum class MetalSupportType : uint8_t
      */
     BoxedCoated,
 };
+constexpr uint8_t MetalSupportTypeCount = 13;
```

Comment:
Should be also moved to the cpp now, nothing else uses that. Also missing static

Reply:
That is true, but having it close to the actual enum avoids situations where the enum is changed and the count is missed. For that reason, I would prefer keeping it here.

Reply:
You could add a Max/Last member to the enum then such mistake won't ever happen.